Add preserve_tone option to autocontrast (#5350)
python-pillow/Pillow: docs/releasenotes/8.2.0.rst

    im.save("out.tif", icc_profile=...)

ImageOps.autocontrast: preserve_tone
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

> The default behaviour of the :py:meth:`~PIL.ImageOps.autocontrast` is to normalize

Reviewer: ```suggestion
The default behaviour of :py:meth:`~PIL.ImageOps.autocontrast` is to normalize
```
Minor suggestion.

> ImageOps.autocontrast: tone preserving option
> ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> 
> The default behaviour of the :py:meth:`~PIL.ImageOps.autocontrast` is to normalize separate
> histograms for each color channel, it changes tone of the image. Added ``preserve_tone``
> argument which avoids changing the tone of the image using one luminance histogram for all
> channels.

Reviewer: ```suggestion
ImageOps.autocontrast: preserve_tone
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

The default behaviour of the :py:meth:`~PIL.ImageOps.autocontrast` is to normalize
separate histograms for each color channel, changing the tone of the image. The new
``preserve_tone`` argument keeps the tone unchanged by using one luminance histogram
for all channels.
```
Suggestion.

Author: Thank you :-)
